binr/rasm2/rasm2.c

```diff
@@ -375,7 +375,8 @@ int main (int argc, char *argv[]) {
 		r_anal_set_bits (anal, sysbits);
 	}
 	
-	r_asm_set_big_endian (a, true);
+	r_asm_set_swap_endian (a, false);
```

Comment:
swap_endian means nothing. it's pretty confusing for users and developers who have to use this api.